### PR TITLE
*: Tiny clean up

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @hanfei19910905 @shenli @coocood

**Comment by shenli** [comment]
LGTM

File: plan/expression_rewriter.go [hunk 1]
```diff
@@ -146,18 +149,21 @@ func (er *expressionRewriter) Leave(inNode ast.Node) (retNode ast.Node, ok bool)
...
-					er.err = errors.Trace(err)
-					return retNode, false
-				}
-				if column != nil {
-					er.correlated = true
-					break
-				}
+		if column != nil {
+			er.ctxStack = append(er.ctxStack, column)
+			break
```
Comment (coocood) [inline plan/expression_rewriter.go]: This change doesn't make the code easier to read.

**Comment by coocood** [comment]
LGTM

**Pull request merged**